huaweicloud/resource_huaweicloud_compute_instance_v2_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/huaweicloud/golangsdk/openstack/compute/v2/extensions/secgroups"
 	"github.com/huaweicloud/golangsdk/openstack/compute/v2/extensions/volumeattach"
 	"github.com/huaweicloud/golangsdk/openstack/compute/v2/servers"
+	"github.com/huaweicloud/golangsdk/openstack/ecs/v1/tags"
```

Comment:
the resource path in "ecs/v1/tags" is "servers" which is out-dated in some region, so we should replace it with "common/tags" with "cloudservers", please reference to #357.